### DISCUSSION TITLE
Collabora failed to connect

### DISCUSSION THREAD

**Discussion by msman0** [nextcloud/all-in-one#4629] [Questions] [answered]
Hi, I have a new Nextcloud AIO installation. Since beginning Collabora didn't work. I get the message:

`Failed to connect to the remote server: cURL error 7: Failed to connect to domain port 443 after 1 ms: Couldn't connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://domain/hosting/discovery`

The container is running in AIO interface.

How can I get working Collabora?
Thank you!

**Comment by Zoey2936** [comment]
See #4440

**Comment by szaimen** [comment, marked as answer]
Hi @msman0 can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by msman0** [comment]
Hi @szaimen thank you very much. I have done the first to remove 127.0.1.1 yourdomain.com from hosts and it work now!
Best regards
